.github/workflows/run_pr_commenter.yml

```diff
@@ -0,0 +1,27 @@
+name: Run PR Commenter
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  run_pr_commenter:
+    name: Run PR Commenter
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: actions/checkout@v4
+      with:
+        ref: main
+        path: target_for_diff
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - run: pip install requests
+
+    - name: Run G Diff
+      id: run-diff
+      run: |
+        python ${{runner.workspace}}\measurementlink-labview\Source\Tools\run_diff.py --token ${{secrets.GITHUB_TOKEN}} --pull-req ${{github.event.pull_request.number}}
```

Comment:
Use forward slashes in GH workflows for portability, unless the app absolutely requires backslashes (Python does not)

Reply:
Updated to forward slashes in latest update.  Also corrected to use forward slashes in  `run_g_tests`